lightning_server.cc

```diff
@@ -54,6 +56,20 @@ void LightningServer::start() {
         continue;
     }
 
+    // DEBUG Testing Below; Can be removed later
```

Comment:
Could we remove this commented-out code? You could save it in a stash or local branch to preserve it, but this lets us keep the codebase clean.